### PR TITLE
Refactor import and export statements

### PR DESCRIPTION
**Pull request by IlyaKhD**

### CODE REVIEW

File: tools/src/helpers.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import * as dasherize from "dasherize";
-import { extname as getPathExtension } from "path";
+import {extname as getPathExtension } from "path";
```
Comment (lukyanovas) [inline tools/src/helpers.ts]: missed space

File: tools/src/generator.ts [hunk 2]
```diff
@@ -1,13 +1,34 @@
...
-    IOption,
-    IPropTyping
+  IComponent,
+  INestedComponent,
+  IOption,
+  IPropTyping
 } from "./component-generator";
+
 import { toPropTypingType } from "./converter";
+import generateIndex, { IReExport} from "./index-generator";
```
Comment (lukyanovas) [inline tools/src/generator.ts]: Missed space

File: src/core/component.ts [hunk 3]
```diff
@@ -1,14 +1,26 @@
...
+} from "./template";
+
+import {
+  IDxTemplate,
+  ITemplateInfo,
+  IWrappedTemplateInfo,
+  TemplateHelper
+} from "./template-helper";
+
+import {
```
Comment (lukyanovas) [inline src/core/component.ts]: I am not sure that's really worth it

File: src/core/widget-config.ts [hunk 4]
```diff
@@ -39,4 +39,6 @@ function separateProps(
     return { options, defaults, templates };
 }
 
-export { separateProps };
+export {
```
Comment (lukyanovas) [inline src/core/widget-config.ts]: Should be default?

File: tools/src/converter.ts [hunk 5]
```diff
@@ -1,5 +1,11 @@
-import { IArrayDescr, ITypeDescr } from "../integration-data-model";
-import { lowercaseFirst } from "./helpers";
+import {
+    IArrayDescr,
+    ITypeDescr
+} from "../integration-data-model";
+
+import {
```
Comment (lukyanovas) [inline tools/src/converter.ts]: Same

File: tools/src/generator.ts [hunk 6]
```diff
@@ -1,13 +1,34 @@
-import { writeFileSync as writeFile } from "fs";
-import { dirname as getDirName, join as joinPaths, relative as getRelativePath, sep as pathSeparator } from "path";
-import { IArrayDescr, IComplexProp, ICustomType, IModel, IProp, ITypeDescr, IWidget } from "../integration-data-model";
+import {
```
Comment (lukyanovas) [inline tools/src/generator.ts]: same

File: src/core/widget-config.ts (hunk 4 above)
Comment (IlyaKhD) [reply src/core/widget-config.ts]: Nope, the file is "widget-config.ts", it would be strange if `default export` was `separateProps `

**Pull request merged**